<launch>
  <arg name="namespace"               default="hanyang_gripper"/>
  <arg name="is_loop"               default="false"/>

  <param name="$(arg namespace)/trajectory_info" value="$(find hanyang_gripper_operator)/config/motion_absolute.yaml"/>
  <param name="$(arg namespace)/initialize_to_max" value="$(find hanyang_gripper_operator)/config/initialize_to_max.yaml"/>
  <param name="$(arg namespace)/finger_fold" value="$(find hanyang_gripper_operator)/config/finger_fold.yaml"/>
  <param name="$(arg namespace)/finger_unfold" value="$(find hanyang_gripper_operator)/config/finger_unfold.yaml"/>
  <param name="$(arg namespace)/grip_to_size" value="$(find hanyang_gripper_operator)/config/grip_to_size.yaml"/>

  <node name="grip_operator" pkg="hanyang_gripper_operator" type="grip_operator"
        required="true" output="screen" ns="hanyang_gripper">
    <param name="is_loop" value="$(arg is_loop)"/>
  </node>
</launch>
